File: sandbox/python/src/TeamSubTeamsDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        response = api.TeamApi(api_client).team_sub_teams(
            team_id="4fea99bfcf2b26bfccf6cea3e127fb8bb74d8d9c",
            page=1,
            page_size=20,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Team#team_sub_teams: %s\n" % e)

File: sandbox/java/src/main/java/com/dropbox/sign_sandbox/TeamSubTeamsDefaultExample.java
package com.dropbox.sign_sandbox;

import com.dropbox.sign.ApiException;
import com.dropbox.sign.Configuration;
import com.dropbox.sign.api.*;
import com.dropbox.sign.auth.*;
import com.dropbox.sign.model.*;

import java.io.File;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class TeamSubTeamsDefaultExample
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();
        config.setUsername("YOUR_API_KEY");
        // config.setAccessToken("YOUR_ACCESS_TOKEN");

        try
        {
            var response = new TeamApi(config).teamSubTeams(
                "4fea99bfcf2b26bfccf6cea3e127fb8bb74d8d9c",
                1,
                20
            );

            System.out.println(response);
        } catch (ApiException e) {
            System.err.println("Exception when calling Team#teamSubTeams");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}
